### PR TITLE
Mk add item styling

### LINKED ISSUE #35: Style <Add Item/> component
Make the /add-item page shine! :) Pending design decisions that we make as a team.

### PR DESCRIPTION
**Pull request by mira-kine**
## Description

Added styling to `AddItem` according to the figma mock up. So far, it is not fully accurate because we need to accept and merge the other pull request with the time frame changes. I created a pull request anyway so that I can receive feedback on the way I have set up my containers, and for any other styling ideas/changes my team may want. I will continue to work on this page after I have merged the other issue, as I expect there to be merge conflicts. 

## Related Issue

Closes #35 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### Before

Screen Shot 2022-08-26 at 4 26 55 PM

### After

Screen Shot 2022-08-26 at 4 26 34 PM

## Testing Steps / QA Criteria

1. From your local machine git checkout main then git pull
2. git checkout MK-add-item-styling
3. npm run start
4. Go to "Add Item" to look for styling

### CODE REVIEW

File: src/views/AddItem.css
Comment (Trevor-Rezac) [inline src/views/AddItem.css:82]: Everything is looking great so far! I think we might just need to change this <p> tag to a different font color. It looks like the text confirmation or error when adding an item is the same color as the background, so I can't see the message that my item was added or is already on the list. Same thing is happening with "Your list name is" on the list page.

**Review by Trevor-Rezac — APPROVED** [review]
Approved with just one minor suggestion to font color!

**Review by michellerenehey — APPROVED** [review]
Such a great start! Good idea to stop before getting too far ahead with merging!

**Review by N-Zubko — APPROVED** [review]
I love to see mockup coming to life 🥲

File: src/views/AddItem.css
Comment (nickytonline) [inline src/views/AddItem.css:17]: Dropping a suggestion for the styling. Currently, the styling of the contents of an item is a bit off-center. You can use `display: flex` on the `.label` to get things to line up better. The spacing gets condensed a bit but you can use e.g. `gap: 10px` as well and they'll space out better.

Using `gap` on the `.timeframe` selector instead of 

```css
.timeframe label {
	padding: 10px;
}
```
also might work better.

https://user-images.githubusercontent.com/833231/187012137-e8761de2-9a2b-4127-9a99-26206a925374.mp4

**Review by nickytonline — COMMENTED** [review]
Left some styling suggestions.

**Review by lindseyindev — APPROVED** [review]
THIS LOOKS SO GOOD! I love all the classes you added and it is responsive.

 I think Nick gave really great feedback on centering the icons to the labels, as I was demoing it out that was something that stood out to me, I approve but would love if that could be incorporated but if there isn't time to get to the flexbox on the label I understand and am super excited to see this styling live on the app!

**Pull request merged**
